docs/sql/mssql/migration_1.6.0_1.7.0.sql

```diff
@@ -12,6 +12,10 @@ GO
 ALTER TABLE pa_activation ALTER COLUMN extras varchar (4000);
 GO
 
+-- Changeset powerauth-java-server/1.7.x/20240115-add-columns-fido2::4::Lubos Racansky
+UPDATE pa_activation SET protocol = 'powerauth' WHERE protocol is null;
```

Comment:
I would recommend, as this is manual work anyway to use

`alter table pa_activation add protocol varchar(255) default 'powerauth' WITH VALUES;
`
It should prevent potential performance issues. (there is no way how to do it in Liquibase via DDL)

Reply:
There is one workaround https://github.com/liquibase/liquibase/issues/4644#issuecomment-2000458786
Since the changeset has been applied, I am afraid that we cannot use it.
But we may use native SQL in the changeset. Or are you speaking only about this SQL script?

Reply:
In this SQL we should use this SQL statement.

Yes, exactly. In LB we need to add SQL update statement - as you have it not. It is fine.